variables.tf

```diff
@@ -0,0 +1,120 @@
+variable "create_instance" {
+  type = bool
+  description = "Shether to create the alicloud_simple_application_server_instance"
+  default = false
+}
+
+variable "create_snapshot" {
+  type = bool
+  description = "Shether to create the alicloud_simple_application_server_snapshot"
+  default = false
+}
+
+variable "create_image" {
+  type = bool
+  description = "Shether to create the alicloud_simple_application_server_custom_image"
+  default = false
+}
+
+variable "instance_id" {
+  type = string
+  description = "the alicloud_simple_application_server_instance ID"
+  default = ""
+}
+
+variable "system_snapshot_id" {
+  type = string
+  description = "the System Snapshot ID"
```

Comment:
An existing simple application server instance id. It will be ignored when `create_snapshot = true`